test/math.jl

```diff
@@ -760,6 +760,35 @@ end
     @test quadgk(cos, 0,0.7,1, norm=abs)[1] ≈ sin(1)
 end
 
+module Test19626
+    using Base.Test
+
+    # Define a mock physical quantity type
+    immutable MockQuantity{T} <: Number
+        val::T
```

Comment:
Is the `<: Number` necessary?  I wouldn't bother parameterizing the type.  Just make `val::Float64`.